examples/atomic-registry/systemd/oauthclient.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
nit: this would generally not have the leading document indicator and the rest of the object would be de-indented one level.